nbextensions/bulkImportCell/bulkImportCell.js

```diff
@@ -704,7 +706,9 @@ define([
          */
         function _switchToTab(tab) {
             state.tab.selected = tab;
-            return stopWidget().then(() => {
+            return Promise.try(() => {
+                stopWidget();
+            }).finally(() => {
```

Comment:
wrap the widget stopping in a `try` to prevent widgets that aren't running from throwing errors if they are stopped.